src/graphql/resolvers/mempool-resolver.ts

```diff
@@ -40,6 +42,11 @@ class UnconfirmedBoxArguments {
 
 @Resolver(Mempool)
 export class MempoolResolver {
+  private nodeservice: nodeService;
```

Comment:
Just a code styling hint: class names should be in PascalCase and properties should be in camelCase.